src/zkevm_specs/evm_circuit/execution/callop.py

```diff
@@ -34,12 +32,10 @@ def callop(instruction: Instruction):
         if is_delegatecall == 1
         else (Word(0), Word(0))
     )
-    parent_caller_address = instruction.word_to_address(parent_caller_address_word)
+    instruction.word_to_address(parent_caller_address_word)
```

Comment:
I'm not sure to understand what you want to do here. Is that an oversight?

Reply:
Nice catch! Fixed in [dec73b2](https://github.com/privacy-scaling-explorations/zkevm-specs/pull/465/commits/dec73b295494a00caf23fba24e42bbad14880fbc)